platforms/jvm/language-java/src/main/java/org/gradle/api/internal/tasks/compile/JdkJavaCompiler.java

```diff
@@ -85,18 +95,14 @@ private JavaCompiler.CompilationTask createCompileTask(JavaCompileSpec spec, Api
                 new CompilationSourceDirs(spec),
                 new CompilationClassBackupService(spec, result)
             );
+            // task classloader becomes: jdk-tools
         }
         Set<AnnotationProcessorDeclaration> annotationProcessors = spec.getEffectiveAnnotationProcessors();
         task = new AnnotationProcessingCompileTask(task, annotationProcessors, spec.getAnnotationProcessorPath(), result.getAnnotationProcessingResult());
         task = new ResourceCleaningCompilationTask(task, fileManager);
         return task;
     }
 
-    private static boolean shouldUseProblemsApiReporting() {
-        String property = System.getProperty("org.gradle.internal.emit-compiler-problems");
-        return Boolean.parseBoolean(property);
-    }
```

Comment:
💭 @donat This was the feature flag control.

Reply:
Cool, thanks for the pointer!